aws/resource_aws_cloudfront_distribution.go

```diff
@@ -432,14 +432,14 @@ func resourceAwsCloudFrontDistribution() *schema.Resource {
 			},
 			"restrictions": {
 				Type:     schema.TypeSet,
-				Required: true,
+				Computed: true,
```

Comment:
Is there a reason you chose `Computed: true` here instead of `Optional: true`? Same with below

Reply:
Just because I thought that it may require computing so that the default could be set to none further down the chain. Reading the aws docs again, it seems that is not the case so I have updated that.